contract_client.go

```diff
@@ -25,6 +25,7 @@ func (api *apiContractClient) Compose(ctx context.Context, space, duration uint6
 		Option("billing-price", options.BillingPrice).
 		Option("billing-period", options.BillingPeriod).
 		Option("percent-approvers", options.PercentApprovers).
+		Option("private-key", options.PrivateKey).
```

Comment:
use crypto.MarshalPrivateKey, then takes those bytes and convert them to hex with `hex.EncodeString`. On the other side(commands) take the string from the option, convert to bytes with `hex.DecodeString` and after convert to PrivateKey with crypto.UnmrshalPrivateKey